src/clarity-angular/datagrid/datagrid-action-overflow.ts

```diff
@@ -61,6 +70,11 @@ export class DatagridActionOverflow {
             //Start checking if current and actionMenuHost are equal. If not traverse to the parentNode and check again.
             while (current) {
                 if (current === actionMenuHost) {
+                    // if this datagrid has checkboxes, uncheck all but this current one
+                    if (this.selection.selectable) {
+                        this.selection.current = [];
+                        this.selection.setSelected(this.item, true);
+                    }
```

Comment:
I had a discussion with @yenmade about this. The wanted behavior from UX point of view is to de-select other selected rows if you click on the ellipses icon (yes, that part is missing in the original spec/doc). I will modify the behavior per your suggestion.